transform/binary-plugin/examples/openshift/cmd.go

```diff
@@ -11,22 +11,29 @@ import (
 
 var logger logrus.FieldLogger
 
+const (
+	// flags
+	StripDefaultPullSecrets = "strip-default-pull-secrets"
+	PullSecretReplacement   = "pull-secret-replacement"
+	RegistryReplacement     = "registry-replacement"
+
+)
 func main() {
 	logger = logrus.New()
 	// TODO: add plumbing for logger in the cli-library and instantiate here
 	fields := []transform.OptionalFields{
                 {
-                        FlagName: "StripDefaultPullSecrets",
-                        Help:     "Whether to strip Pod and BuildConfig default pull secrets (beginning with builder/default/deployer-dockercfg-) that aren't replaced by the map param PullSecretReplacement",
+                        FlagName: StripDefaultPullSecrets,
+                        Help:     "Whether to strip Pod and BuildConfig default pull secrets (beginning with builder/default/deployer-dockercfg-) that aren't replaced by the map param pull-secret-replacement",
```

Comment:
```suggestion
                        Help:     "Whether to strip Pod and BuildConfig default pull secrets (beginning with builder/default/deployer-dockercfg-) that aren't replaced by the map param" + PullSecretReplacement,
```

That'll make it consistent with other help messages included in this PR.

Reply:
Fixed